app/Models/Campaign.php

```diff
@@ -430,6 +430,13 @@ public function getDefaultGalleryVisibilityAttribute(): mixed
         return Arr::get($this->settings, 'gallery_visibility', 'all');
     }
 
+    /**
+     */
+    public function getNamePrivateEntities(): mixed
```

Comment:
We name all of these helper classes with get...Attribute to make use of Laravel's attribute mapper, that was we can just call name_private_entities. The typing should also be a bool rather than mixed